crates/cheatnet/src/execution/calls.rs

```diff
@@ -84,3 +84,4 @@ pub fn execute_library_call(
 
     execute_inner_call(&mut entry_point, vm, syscall_handler, remaining_gas)
 }
+
```

Comment:
Shouldn't be changed

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/589/commits/311737c203573f3bab0ae407a2a096fb8b7d5a6a